ext/drmgr/drmgr.c

```diff
@@ -667,11 +667,6 @@ drmgr_bb_event(void *drcontext, void *tag, instrlist_t *bb,
             }
             /* XXX: add checks that cb followed the rules */
         }
-        /* XXX i#1723: in f28be26, we added auto-predication of instrumentation
-         * in Thumb IT blocks here, but it was asymmetric wrt ARM mode.
-         * To re-add the feature, we should have it apply to both Thumb and ARM,
-         * and have it controllable by the client.
-         */
```

Comment:
Shouldn't we leave this until we make the new approach automated here?  Maybe just update this comment to mention it?